### ISSUE TITLE
Accounts

### ISSUE THREAD

**Issue by Edward-Stryfe** [fastcast/fastcast#13]
Now I can see how you get the videos posted through an HTML form and PHP script but come time to make accounts and track information, like say likes and dislikes or number of times watched, how do you plan on doing things? Will it be more 4chan esc where you just post a new video and fill out a form and it just asks for a username each time and makes it otherwise anonymous. or do you have database method in mind?

**Comment by jakefb** [comment]
Hi @Edward-Stryfe, I'm planning on using django to allow users to define their own torrents. All the torrents will be stored in a postgres database. The html version of this project will probably be renamed fastcast-static and kept in it's own repository for people who prefer to use that. This repository will host the django code. 

Have you got any experience using django / python? It would be great to have your help!

**Comment by Edward-Stryfe** [comment]
I wish I did I may need to pick it up though cause I'm working on setting up a project similar to this so ATM I need to learn more JS but I am probably going to have to use the same solution though some sources say that it's possible to manage SOME MySQL through JS so I may go that way or do something like 4chan/PirateBay esc so that there isn't any database of anything and theres just a form anyone can use to submit magnet links and a few other inputs and the rest autofill. 

Cause I'm terrible with databases even with PHP and MySQL I'm newb as can be. And I am working on learning Assembly so if I find the time I'll try to pick up Python.

**Comment by jakefb** [comment]
Django is a MVC so you don't need much database knowledge to use it - it manages the SQL database for you. It's pretty simple once you learn a bit about how to use it and python is a lot more elegant than PHP. You should do the django tutorial here https://docs.djangoproject.com/en/1.10/intro/tutorial01/

Let me know how it goes and I'm available to help if you need!

**Comment by Edward-Stryfe** [comment]
Alright and is Python a web language that can be used in place of PHP and if so is it globally supported? And I'll be sure to ring you up via email or gitter if I need a hand thanks for that.

**Comment by jakefb** [comment]
Yeah python is very popular, it's a general purpose interpreted language but is also used a lot on servers and has many packages that you can use so you don't have to reinvent the wheel.

**Comment by Edward-Stryfe** [comment]
Nice would it work for form scripts like PHP does?

**Comment by jakefb** [comment]
Django works really well with forms https://docs.djangoproject.com/en/1.10/topics/forms/ It has CSRF protection too.

There a few ways you can create forms, you can write a form class or even easier if you would like to manipulate a model in the database you can automatically create a model form with generic views https://docs.djangoproject.com/en/1.10/topics/class-based-views/generic-editing/
